src/jit/codegenxarch.cpp

```diff
@@ -2687,18 +2712,19 @@ void CodeGen::genLclHeap(GenTree* tree)
             {
                 inst_IV(INS_push_hide, 0); // push_hide means don't track the stack
             }
+
+            lastTouchDelta = 0;
+
             goto ALLOC_DONE;
         }
 
         bool doNoInitLessThanOnePageAlloc =
```

Comment:
I realize that this is pre-existing, but this negative name makes the negation of it below even more confusing than this already is. Also, could you add a comment to this? Is it intended to be "doNot" rather than "doNo"?

Reply:
It's supposed to be parsed "do" "no-init", as in "we don't need to zero init the allocated memory". I'll reverse the condition and name it "initMemOrLargeAlloc"

Reply:
I think that would be clearer, thanks.